git_day_1.adoc

```diff
@@ -0,0 +1,138 @@
+Git Gud Speaker Outline - Day 1
+-------------------------------
+:hardbreaks:
+
+The Basics
+~~~~~~~~~~
```

Comment:
I've never seen these style of headers in asciidoc.

Usually, I think the standard is to use a number of `=`s where the amount is the heading level.

```suggestion
== The Basics
```

Reply:
I was using an aciidoc cheat sheet I found somewhere on google, but I do like the == signs more. Fixed.